test/forms/ConvertedPackagingCreditFormProviderSpec.scala

```diff
@@ -26,7 +26,7 @@ class ConvertedPackagingCreditFormProviderSpec extends DecimalFieldBehaviours {
 
   val packagingCredit = 1.25
   val formProvider            = new ConvertedPackagingCreditFormProvider
-  val form: Form[BigDecimal] = formProvider(BigDecimal(99999999.99))
+  val form: Form[BigDecimal] = formProvider(Some(BigDecimal(99999999.99)))
```

Comment:
this covers both branches of testing an should still pass :D 
```suggestion
  val form: Form[BigDecimal] = formProvider(None)
```

Reply:
Seems happy